openedx/features/course_card/views.py

```diff
@@ -1,20 +1,23 @@
 from datetime import datetime
 
 import pytz
+from django.views.decorators.csrf import csrf_exempt
+
 from course_action_state.models import CourseRerunState
-from philu_overrides.helpers import get_user_current_enrolled_class
 from edxmako.shortcuts import render_to_response
-from openedx.features.course_card.models import CourseCard
-from django.views.decorators.csrf import csrf_exempt
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from openedx.features.course_card.models import CourseCard
+from philu_overrides.helpers import get_user_current_enrolled_class
 from student.models import CourseEnrollment
+
+from ...core.djangoapps.catalog.utils import get_programs
+from ..specializations.helpers import date_from_str
 from .helpers import get_course_open_date
 
 utc = pytz.UTC
 
 
 def get_course_start_date(course):
-
     """
     this function takes course and returns start date of the course
```

Comment:
not part of my changes so not sure if I should make these changes. @saad-waqar thoughts?

Reply:
sorry I didn't realise it wasn't part of your changes.

Reply:
@monofal It's good to not change anything which is not part of the scope to avoid going out of the scope and increasing testing effort, but here if we have noticed something which we can improve it's not much effort and also no need to test it. So I'd also request to please change it.